test/rendering-test/node-rendering.js

```diff
@@ -34,26 +32,32 @@ async function getColor(page, selector) {
 }
 
 async function validateRendering(child) {
-  const browser = await puppeteer.launch({
-    headless: false,
-    executablePath: '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome'
-  });
-  const page = await browser.newPage();
-  await page.waitFor(1000);
-  await page.goto('http://localhost:8080');
-  await page.setViewport({width: 1550, height: 850});
+  try {
+    const browser = await puppeteer.launch({
+      headless: false,
+      executablePath: '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome'
+    });
+    const page = await browser.newPage();
+    await page.waitFor(1000);
+    await page.goto('http://localhost:8080');
+    await page.setViewport({width: 1550, height: 850});
 
-  const color = await getColor(page, 'body > div:nth-child(7) > p');
+    const color = await getColor(page, 'body > div:nth-child(7) > p');
 
-  child.kill();
-  await page.waitFor(1000);
-  await browser.close();
+    child.kill();
+    await page.waitFor(1000);
+    await browser.close();
 
-  if (color !== 'rgb(11, 255, 28)') {
-    console.log('Rendering test failed!');
+    if (color !== 'rgb(11, 255, 28)') {
+      console.log('Rendering test failed!');
+      return false;
+    }
+    return true;
+  } catch (error) {
+    console.log(error);
```

Comment:
Add a console statement that this `works best with Chrome 64`?

Reply:
Yeah, that's a good suggestion, will hook that error, need more testing.

Reply:
I don't think you need any logic or if statements, you can just print that line (in addition to the actual error) whenever an exception happens. Most people will get the exception for that reason, so it makes sense.

Reply:
There are a lot of other exceptions and pretty common, put the "chrome 64" prompt for all the cases could be confusing, for example when one testing cases failed, the browser pauses on that cases, user has to close the browser manually, at the same time "target closed" exception occurs. Similar thing happens if webpack server doesn't set up correctly because of port occupied.
I will handle the "chrome 64" exception in a different PR, cause this fix need be landed ASAP.